pom.xml

```diff
@@ -1,52 +1,61 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<build>
-		<plugins>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-compiler-plugin</artifactId>
-				<version>3.10.1</version>
-				<configuration>
-					<release>8</release>
-					<showDeprecation>true</showDeprecation>
-					<showWarnings>true</showWarnings>
-					<source>10</source>
-					<target>10</target>
-				</configuration>
-			</plugin>
-			<plugin>
-				<groupId>org.jenkins-ci.tools</groupId>
-				<artifactId>maven-hpi-plugin</artifactId>
-				<version>3.25</version>
-				<configuration>
-					<minimumJavaVersion>8</minimumJavaVersion>
-				</configuration>
-			</plugin>
-		</plugins>
-	</build>
+
 	<parent>
 		<groupId>org.jenkins-ci.plugins</groupId>
 		<artifactId>plugin</artifactId>
-		<version>4.44</version>
+		<version>4.48</version>
 		<relativePath />
 	</parent>
+
 	<groupId>io.jenkins.plugins</groupId>
 	<artifactId>crowdstrike-security</artifactId>
+	<version>${changelist}</version>
 	<packaging>hpi</packaging>
+
 	<name>CrowdStrike Security</name>
 	<description>
-		Enables assessment of Docker container images for vulnerabilities by sending them to CrowdStrike's Falcon Image Assessment module, pulling the report of vulnerabilities found.
+		Enables assessment of Docker container images for vulnerabilities by sending them to CrowdStrike's Falcon
+		Image Assessment module, pulling the report of vulnerabilities found.
 	</description>
 	<url>https://github.com/jenkinsci/crowdstrike-security-plugin</url>
+	<licenses>
+		<license>
+			<name>Apache License, Version 2.0</name>
+			<url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+			<distribution>repo</distribution>
+			<comments>A business-friendly OSS license</comments>
+		</license>
+	</licenses>
 
 	<scm>
-		<connection>scm:git:ssh://github.com/jenkinsci/crowdstrike-security-plugin.git</connection>
-		<developerConnection>scm:git:ssh://git@github.com/jenkinsci/crowdstrike-security-plugin.git</developerConnection>
-		<url>https://github.com/jenkinsci/crowdstrike-security-plugin</url>
-		<tag>HEAD</tag>
+		<connection>scm:git:ssh://github.com/${gitHubRepo}.git</connection>
+		<developerConnection>scm:git:ssh://git@github.com/${gitHubRepo}.git</developerConnection>
+		<url>https://github.com/${gitHubRepo}</url>
+		<tag>${scmTag}</tag>
 	</scm>
-	<version>1.0.1-SNAPSHOT</version>
+	<distributionManagement>
+		<repository>
+			<id>maven.jenkins-ci.org</id>
+			<url>https://repo.jenkins-ci.org/releases/</url>
+		</repository>
+		<snapshotRepository>
+			<id>maven.jenkins-ci.org</id>
+			<url>https://repo.jenkins-ci.org/snapshots/</url>
+		</snapshotRepository>
+	</distributionManagement>
+
+	<properties>
+		<changelist>99999-SNAPSHOT</changelist>
```

Comment:
This name seems incorrect.

Reply:
No, it's part of the JEP-229. Please see https://www.jenkins.io/doc/developer/publishing/releasing-cd/